debian/control

```diff
@@ -9,8 +9,8 @@ Build-Depends: bash-completion,
                fakeroot,
                gettext,
                gnupg2,
+               golang (>=2:1.6) | golang-1.6,
                golang-check.v1-dev,
-               golang-go,
```

Comment:
You're removing this here and then adding /usr/lib/go* to the path explicitly below. Sounds like something isn't quite right.